pkg/replicator/replicator.go

```diff
@@ -25,6 +25,7 @@ type Replicator interface {
 	StartReplication(host, replicaPass string, port int32) error
 	ReplicationStatus() (ReplicationStatus, string, error)
 	EnableReadonly() error
```

Comment:
As I remember if you set SRO the RO also is enabled. Should we have it?

Reply:
line 29 - should we check IsSuperReadonly?

Reply:
`EnableReadOnly` is not used anywhere, removed it and check for `super_read_only`: https://github.com/percona/percona-server-mysql-operator/pull/87/commits/6a0bbc701c24196a881a66146affb31b6ced31b8